store/rootmultistore.go

```diff
@@ -322,10 +319,23 @@ func (rs *rootMultiStore) loadCommitStoreFromParams(id CommitID, params storePar
 		// TODO: id?
 		// return NewCommitMultiStore(db, id)
 	case sdk.StoreTypeIAVL:
+		_, ok := key.(*sdk.KVStoreKey)
```

Comment:
Definitely out of scope of this PR, but I really think we should avoid using named return variables in large and/or complex functions/methods.